src/query/api/v1/handler/prometheus/remote/write.go

```diff
@@ -144,6 +162,7 @@ func NewPromWriteHandler(options options.HandlerOptions) (http.Handler, error) {
 		forwardHTTPClient:      xhttp.NewHTTPClient(forwardHTTPOpts),
 		forwardingBoundWorkers: forwardingBoundWorkers,
 		forwardContext:         context.Background(),
+		forwardRetrier:         retry.NewRetrier(forwardRetryConfig.NewOptions(scope)),
```

Comment:
Can you subscope this scope? Just so it doesn't overwrite other retry metrics.  Probably just `scope.SubScope("forwarding-retry")`